resources/measures/set_primary_kitchen_equipment/measure.rb

```diff
@@ -188,13 +188,28 @@ def run(model, runner, user_arguments)
     # make hash of equipment quantities
     appliance_quantity_hash = Hash[li_appliance_types.zip li_appliance_quantities]
     # make list of electric equipment power (in kW)
-    li_appliance_electric_power_kW = [11.000,12.000,22.000,15.000,12.000,15.000]
+    li_appliance_electric_power_kW = [10.815,17.116,14.009,12.104,21.014,26.964]
```

Comment:
Weren't these changes all reflected in a separate PR: https://github.com/NREL/ComStock/pull/66